utils/wasm-gen/src/config/syscalls/param.rs

```diff
@@ -71,7 +71,7 @@ impl SyscallsParamsConfig {
 
         // Setting regular params rules.
         self.with_rule(Length, (0..=1600).into())
-            .with_rule(Gas, (0..=250_000_000_000).into())
+            .with_rule(Gas, (0..=10_000_000_000).into())
```

Comment:
Somehow
```suggestion
            .with_rule(Gas, (0 || MAILBOX_THRESHOLD..=10_000_000_000).into())
```

Reply:
or just avoid 0 - but it seems incorrect
the thing is the Gas type describes both reservation amount and gas to be applied for sending - maybe its worth to separate to be more specific? like gas for sending may be 0 and its common case, while zero reservation leads to an error

Reply:
Ok, makes sense
But anyway we need to increase possibility to call send with 0 gas as well